ale/drivers/dawn_driver.py

```diff
@@ -0,0 +1,126 @@
+import ale
+from ale.drivers.base import *
+from ale import config
+
+config.dawn = '/data/fast/kernels/dawn-m_a-spice-6-v1.0/dawnsp_1000/extras/mk'
+
+class DawnCamera(Driver, Framer, PDS3, Spice, RadialDistortion):
+    """
+    
+    """
+    
+    @property
+    def metakernel(self):
+        """
+        Returns latest instrument metakernels
+
+        Returns
+        -------
+        : string
+          Path to latest metakernel file
+        """
+        metakernel_dir = config.dawn
+        mks = sorted(glob(os.path.join(metakernel_dir,'*.tm')))
+        if not hasattr(self, '_metakernel'):
+            for mk in mks:
+                if str(self.start_time.year) in os.path.basename(mk):
+                    self._metakernel = mk
+        return self._metakernel
+        
+    @property
+    def instrument_id(self):
+        """
+        Returns an instrument id for uniquely identifying the instrument, but often
+        also used to be piped into Spice Kernels to acquire IKIDs. Therefore the
+        the same ID that Spice expects in bods2c calls.
+
+        Returns
+        -------
+        : str
+          instrument id
+        """
+        instrument_id = label["INSTRUMENT_ID"]
+        filter_number = label["FILTER_NUMBER"]
+        
+        return "DAWN_{}_FILTER_{}".format(instrument_id, filter_number)
+    
+    @property
+    def _odtk(self):
+        """
+        Returns
+        -------
+        : list
+          Radial distortion coefficients
+        """
+        return spice.gdpool('INS{}_RAD_DIST_COEFF'.format(self.ikid),0, 1).tolist()
+    
+    @property
+    def label(self):
+        """
+        Loads a PVL from from the _file attribute and
+        parses the binary table data.
+
+        Returns
+        -------
+        PVLModule :
+            Dict-like object with PVL keys
+        """
+        if not hasattr(self, "_label"):
+            if isinstance(self._file, pvl.PVLModule):
+                self._label = self._file
+            try:
+                self._file.replace("\\", "/")
+                self._label = pvl.loads(self._file)
+            except Exception:
+                slabel = open(self._file).read().replace("\\", "/")
+                self._label = pvl.loads(slabel)
+            except:
+                raise ValueError("{} is not a valid label".format(self._file))
+        return self._label
+    
+    @property
+    def spacecraft_name(self):
+        """
+        Spacecraft name used in various Spice calls to acquire
+        ephemeris data.
+
+        Returns
+        -------
+        : str
+          Spacecraft name
+        """
+        return 'DAWN'
```

Comment:
I am more comfortable with this than the target (below) because the spacecraft does not have the potential to change mid-mission (I would hope...but who knows!).